python/README.md

```diff
@@ -10,6 +12,7 @@ The images below include variations for only the core packages in the [Intel® D
 | ---------------------- | ---------- |
 | `3.10-full`, `latest`  | `2024.2.0` |
 | `3.10-core`            | `2024.2.0` |
+| `3.10-xpu-idp`         | `2024.2.1` |
```

Comment:
Maybe just `3.10-xpu`? Do we have both full and core options for the package? (see overrides file)

Reply:
support for `core` and `full` added